google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryDependencyWriter.java

```diff
@@ -121,7 +123,14 @@ protected void run(@NotNull Result result) throws Throwable {
         List<MavenId> newMavenIds = mavenIdsMap.get(false);
         if (!newMavenIds.isEmpty()) {
           newMavenIds.forEach(
-              mavenId -> MavenDomUtil.createDomDependency(model, /* editor= */ null, mavenId));
+              mavenId -> {
+                MavenDomUtil.createDomDependency(model, /* editor= */ null, mavenId);
+                UsageTrackerProvider.getInstance()
+                    .trackEvent(GctTracking.CLIENT_LIBRARY_ADD_LIBRARY)
+                    .addMetadata("build", "maven")
```

Comment:
can we make these variables: e.g. GctTracking.METADATA_BUILD_SYSTEM_KEY, GctTracking.METADATA_BUILD_SYSTEM_MAVEN or something like that